tests/fault_tolerance/migration/test_sglang.py

```
@@ -187,12 +191,14 @@ def __init__(
-        try:
-            # system_port is a required parameter, always set in __init__
-            deallocate_port(self.system_port)
-        except Exception as e:
-            logging.warning(f"Failed to release SGLang worker port: {e}")
+        """Release allocated ports when worker exits."""
+        for port in (self.system_port, self.bootstrap_port, self.prefill_port):
+            if port is None:
+                continue
+            try:
+                deallocate_port(port)
+            except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)